packages/services/api/src/modules/schema/providers/inspector.ts

```diff
@@ -65,7 +64,7 @@ export class Inspector {
                 const fieldAfter = typeAfter.getFields()[field];
                 // Adding a non-nullable input field to a used input object type is a breaking change.
                 // That's why we need to check if the input type is used, not the field itself (as it's new)
-                if (change.type === ChangeType.InputFieldAdded && isNonNullType(fieldAfter.type)) {
```

Comment:
@TuvalSimha this is not the same, that's why the integration test fails.

Hive is using this version of Inspector, alpha snapshot: https://github.com/kamilkisiela/graphql-inspector/pull/2673

I think we can do one of the two things here:
a) merge my PR as it's now proven to be working and useful in Hive
b) release an alpha snapshot that contains both, you and mine changes to Inspector

It's up to you to decide :)